### PR TITLE
Added a method to check if a string contains only whitespace and acco…

### PR DESCRIPTION
**Pull request by nservidio**
…mpanying test.

### CODE REVIEW

**Comment by loganwright** [comment]
Looks good to me :shipit:

File: SwiftWisdom/Core/NSData/String+NSData.swift [hunk 1]
```diff
@@ -71,6 +71,10 @@ extension String {
     public var ip_length: Int {
         return self.characters.count
     }
+    
+    public var ip_containsOnlyWhitespace: Bool {
+        return self.stringByTrimmingCharactersInSet(NSCharacterSet.whitespaceCharacterSet()).ip_length == 0
```
Comment (wingchi) [inline SwiftWisdom/Core/NSData/String+NSData.swift]: no newlines? `whitespaceAndNewlineCharacterSet()`

File: SwiftWisdom/Core/NSData/String+NSData.swift [hunk 2]
```diff
@@ -71,6 +71,10 @@ extension String {
     public var ip_length: Int {
         return self.characters.count
     }
+    
+    public var ip_containsOnlyWhitespace: Bool {
+        return self.stringByTrimmingCharactersInSet(NSCharacterSet.whitespaceAndNewlineCharacterSet()).ip_length == 0
+    }
 }
```
Comment (Benuuu) [inline SwiftWisdom/Core/NSData/String+NSData.swift:78]: Look at String+Empty.swift

**Comment by Benuuu** [comment]
This also shouldn't exist in String+NSData

**Comment by Benuuu** [comment]
👎

**Pull request closed without merging**